doc/run_on_kubernetes.md

```diff
@@ -63,8 +57,81 @@ Docker image](/doc/build.md).
+and easy to scale up/down the SQLFlow gRPC server according to workload.
+
+1. Build the SQLFlow Hub Docker image and push to a registry server that the Kubernetes Note can access it.
+    ``` bash
+    $ cd k8s/sqlflowhub
+    $ docker build -t <your-repo>/sqlflowhub .
+    $ docker push <your-repo>/sqlflowhub
+    ```
+
+1. Deploy the MySQL, SQLFlow gRPC server and JupyterHub step by step:
+
+    ``` bash
+    kubectl create -f k8s/sqlflow-mysql.yaml
+    kubectl create -f k8s/sqlflow-server.yaml
+    kubectl create -f k8s/sqlflow-jhub.yaml
+    ```
+
+1. Check the SQLFlow Pods, you can find:
+    - A MySQL Pod named `sqlflow-mysql-*`.
+    - A JupyterHub Pod named `sqlflow-jhub-*`.
+    - 3 SQLFlow gRPC server Pods named `sqlflow-server-*`, and it's easy to scale up/down the replica count by modifying the `replicas` field of the yaml file: [k8s/sqlflow-server.yaml](/doc/k8s/sqlflow-server.yaml).
+    ``` bash
+    $ kubectl get pods
+    NAME                              READY   STATUS    RESTARTS   AGE
+    sqlflow-jhub-78f96dcf88-sbvt6     1/1     Running   0          4m13s
+    sqlflow-mysql-55db79fd98-nhfjp    1/1     Running   0          4h8m
+    sqlflow-server-7444b4466d-frbcn   1/1     Running   0          4h
+    sqlflow-server-7444b4466d-h5w9c   1/1     Running   0          4h
+    sqlflow-server-7444b4466d-kndwx   1/1     Running   0          4h
+    ```
+
+1.  Check the SQLFlow Service, so the Notebook server can connect them across their ClusterIP and Port:
+    - A MySQL Service named `sqlflow-mysql`, and
+    - An SQLFlow server Service named `sqlflow-server`
+
+    ``` bash
+    $ kubectl get svc
+    NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)     AGE
+    kubernetes       ClusterIP   10.96.0.1        <none>        443/TCP     29d
+    sqlflow-mysql    ClusterIP   10.102.193.217   <none>        3306/TCP    6h4m
+    sqlflow-server   ClusterIP   10.102.65.39     <none>        50051/TCP   5h56m
+    ```
+
+### Login the JupyterHub
+
+JupyterHub using the PAMAuthenticator as the default authenticate method. the PAM can authenticate the system users with their username and password. You can find more information on [authenticators-users-basics](https://jupyterhub.readthedocs.io/en/stable/getting-started/authenticators-users-basics.html), and other authenticator methods from [here](https://github.com/jupyterhub/jupyterhub/wiki/Authenticators)
+
+Next, please do as the following steps to create a user on the system and log in the Jupyterhub:
+
+1. List the Pods and execute into the `sqlflow-jhub` Pod
+    ``` bash
+    $ kubectl get po
+    NAME                              READY   STATUS    RESTARTS   AGE
+    sqlflow-jhub-78f96dcf88-gp8dg     1/1     Running   0          26m
+    sqlflow-mysql-55db79fd98-nhfjp    1/1     Running   0          51m
+    ...
+    $ kubectl exec -it sqlflow-jhub-78f96dcf88-gp8dg bash 
+    ```
+
+1. Create a user and set a password by the `adduser` command:
```

Comment:
To avoid this additional user creation step, you may wanna config `dummyauthenticator` by https://github.com/jupyterhub/kubespawner/blob/master/jupyterhub_config.py#L32 .

Please remember to install it using `pip install jupyterhub-dummyauthenticator`
[ref](https://github.com/jupyterhub/dummyauthenticator)

Reply:
I'm not sure we can use the DummyAuthenticator to demonstrate the SQLFlow on k8s, seems PAMAuthenticator is more similar to the production environment.